### PR TITLE
Fix signatures for functions with typedef params (#1137).

### PR DESCRIPTION
**Pull request by pq**
Fixes https://github.com/dart-lang/dartdoc/issues/1137

### CODE REVIEW

File: testing/test_package_docs/index.html [hunk 1]
```diff
@@ -84,11 +84,11 @@ <h1 class="title">
 <p>This is an amazing package.</p>
 <p>For example, it:</p><ul><li>Is very fast</li><li>Has zero bugs</li><li>Is free</li></ul>
 <p>It also has some awesome code</p>
-<pre class="prettyprint language-dart"><code class="language-dart">void main() {
+<pre class="dart prettyprint"><code>void main() {
```
Comment (pq) [inline testing/test_package_docs/index.html]: Not sure why this changed?

I did re-gen the docs (`` pub global run grinder update-test-package-docs``) but I don't see how my change could have affected this bit.  Maybe this is left over from another change?

File: testing/test_package_docs/code_in_comments/code_in_comments-library.html [hunk 2]
```diff
@@ -83,11 +83,11 @@ <h1 class="title">
   <div class="col-xs-12 col-sm-9 col-md-8 main-content">
 
     <section class="desc markdown">
-      <pre class="prettyprint language-dart"><code class="language-dart">void main() {
+      <pre class="dart prettyprint"><code>void main() {
```
Comment (devoncarew) [inline testing/test_package_docs/code_in_comments/code_in_comments-library.html]: How did this guy change? Perhaps run `pub get` and re-gen the docs?

Comment (devoncarew) [inline testing/test_package_docs/code_in_comments/code_in_comments-library.html]: I think you need the newer version of markdown.

Comment (pq) [inline testing/test_package_docs/code_in_comments/code_in_comments-library.html]: Yep.  Update fixed it.  Thanks!

**Pull request merged**